tpot/gp_deap.py

```diff
@@ -103,7 +103,8 @@ def varOr(population, toolbox, lambda_, cxpb, mutpb):
 
 
 def eaMuPlusLambda(population, toolbox, mu, lambda_, cxpb, mutpb, ngen, pbar,
-                   stats=None, halloffame=None, verbose=0, max_time_mins=None):
+                   stats=None, halloffame=None, verbose=0, max_time_mins=None,
+                   periodic_pipeline_saver=None):
```

Comment:
Great idea,
in fact in eaMuPlusLambda context this is a general parameter "per generation function"
renaming to per_generation_hook_func to be more explicit in its role.

This can later be used for more extended logic too.